### DISCUSSION TITLE
[HELP]does the api server running?

### DISCUSSION THREAD

**Discussion by Qboy-1214** [waylaidwanderer/node-chatgpt-api#314] [Help]
![image](https://user-images.githubusercontent.com/106801052/232800100-98713100-e791-42c7-98b5-d8e7f9117543.png)
look this pic.
I do this `npm run server`,but the application is stop here.

**Comment by NymphQAQ** [comment]
This is the successful startup.You can try accessing some interfaces and view server.js
